### PR TITLE
remove Translation.lua and add locales for all languages + fix ped has props

### PR DESCRIPTION
**Pull request by enzo2991**

### CODE REVIEW

File: client/Emote.lua [hunk 1]
```diff
@@ -5,6 +5,7 @@ local ChosenDict = ""
 local ChosenAnimOptions = false
 local MovementType = 0
 local PlayerGender = "male"
+local PedHasProp = false
```
Comment (alberttheprince) [inline client/Emote.lua:8]: Just want to know why this was added, doesn't seem related to translations

Comment (alberttheprince) [reply client/Emote.lua:8]: Or was this missing from previous PR?

**Comment by alberttheprince** [comment]
I think as well you may have pulled this translation logic from ESX? Let's give a credit to them in the credits for using this/part of this if you did. 👍 

 https://github.com/esx-framework/esx_core/blob/c80916c3e3b14e3cfce6102801b382e59c49bd27/%5Bcore%5D/es_extended/locale.lua#L3

**Comment by alberttheprince** [comment]
Main question/concern here I have is what is the reasoning for replacing the current system? Could we not split it using the previous system? 

On board to replace if it provides some benefit/there's a goal with this rewrite.

**Comment by enzo2991** [comment]
> Main question/concern here I have is what is the reasoning for replacing the current system? Could we not split it using the previous system?
> 
> On board to replace if it provides some benefit/there's a goal with this rewrite.

The goal is to make it cleaner, if we want to add a language we just have to create a file in the locales folder instead of having to search for its language and our old system doesn't take English by default if the translation doesn't exist in translation.lua

File: client/Emote.lua (hunk 1 above)
Comment (enzo2991) [reply client/Emote.lua:8]: This modification had to be made because the ped preview used the player's preview for the props
